readme.txt

```diff
@@ -3,6 +3,7 @@ Contributors: wordpressdotorg
 Requires at least: 6.4
 Tested up to: 6.4
 Requires PHP: 7.0
+Stable tag: 1.0
```

Comment:
We need to also updte the style.css file and the changelog, can you please update it? Using TT3 as a reference works

Reply:
@MaggieCabrera 

Style.css and change log are up to date

Just Stable tag is missing so I have added